tests/functional/features/chart_source_local.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Chart local source verification
+    Partners or redhat or community can verify their charts by running the
+    chart verifier against a local chart source.
+
+    Scenario: The partner IBM succesfully verifies their chart source using the chart verifier docker image
+        Given: I am a Red Hat partner from IBM
```

Comment:
Sorry that I used colon in my example because of markdown. But `Given` `And` `When` and `Then` sections do not have colons: https://github.com/pytest-dev/pytest-bdd#example